src/versions.js

```diff
@@ -9,6 +9,18 @@ const versions = {
                     designLibraryDoc: '', // TODO: Add Sketch Cloud Link
                 },
             },
+            navigation: {
+                breadcrumbs: {
+                    devLibraryVersion: '1.0.0',
+                    designLibraryVersion: '', // TODO: Add Design Library version
+                    designLibraryDoc: '', // TODO: Add Sketch Cloud Link
+                },
+                'sectional-tabs': {
```

Comment:
Actually this maybe should be `sectionalTabs` instead of `'sectional-tabs'`.  I think we are able to translate between the two casings.  I ran into this same concern for `durationPicker` above and had updated the `<ComponentVersion>` component so it can translate from `'duration-picker'` => `'durationPicker'`.

See https://github.com/saddlebackdev/react-cm-ui/blob/92ee5295c885d9f78203da24b64264029870a7bd/docs/src/global/componentVersionIdentifier.jsx#L24-L30

Reply:
Renamed to `sectionalTabs`